src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -208,6 +208,23 @@ private static (RecalcEngine engine, RecordValue parameters) MutationFunctionsTe
                 recordWithRecord3
             });
 
+            var rValueType = RecordType.Empty()
```

Comment:
>var [](http://example.com/codeflow?start=12&length=3)

This PR may provide an easy way to add these tests, without having to modify C# :  #926